### PR TITLE
add translation to the fallback organizer title here

### PR DESCRIPTION
**Pull request by ggwicz**
a small fix – just an untranslated string in API class.

Came from a .org thread: https://wordpress.org/support/topic/missing-translation-2?replies=5

See: https://central.tri.be/issues/40036

### CODE REVIEW

**Comment by bordoni** [comment]
:8ball:

**Comment by zbtirrell** [comment]
Missing ticket reference
> via jenkins/no-ticket-ref

**Comment by zbtirrell** [comment]
Missing ticket reference
> via jenkins/ticket-ref-check

**Comment by zbtirrell** [comment]
Missing ticket reference
> via jenkins/ticket-ref-check

**Comment by zbtirrell** [comment]
Status|Message|Line
:-----:|-----|-----
:x: | Comma required after last value in array declaration | [API.php:22](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L22)
:x: | Comma required after last value in array declaration | [API.php:29](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L29)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:45](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L45)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:64](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L64)
:warning: | The method parameter $post is never used | [API.php:288](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L288)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:294](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L294)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:299](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L299)
:warning: | The method parameter $post is never used | [API.php:312](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L312)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:322](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L322)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:331](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L331)
:x: | There must be a single space after the first square bracket | [API.php:377](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L377)
:x: | There must be a single space before the last square bracket | [API.php:377](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L377)
:x: | There must be a single space after the first square bracket | [API.php:377](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L377)
:x: | There must be a single space before the last square bracket | [API.php:377](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L377)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:406](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L406)
:x: | String &quot;Unnamed Venue&quot; does not require double quotes; use single quotes instead | [API.php:436](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L436)
:x: | There must be a single space after the first square bracket | [API.php:468](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L468)
:x: | There must be a single space before the last square bracket | [API.php:468](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L468)
:x: | There must be a single space after the first square bracket | [API.php:468](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L468)
:x: | There must be a single space before the last square bracket | [API.php:468](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L468)
:x: | Must use &quot;self::&quot; for local static member reference | [API.php:488](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L488)
:warning: | Comment refers to a TODO task &quot;We should probably do away with 'StateProvince' and stick to 'State' and 'Province'&quot; | [API.php:513](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L513)
:x: | String &quot;United States&quot; does not require double quotes; use single quotes instead | [API.php:516](https://github.com/moderntribe/the-events-calendar/blob/583b0cf024e555639e6fa28e5745ce2d40cbbf43/src/Tribe/API.php#L516)

> via jenkins/codesniffer-phpcs

File: src/Tribe/API.php [hunk 1]
```diff
@@ -343,8 +343,10 @@ private static function saveEventVenue( $data, $post = null, $post_status = 'pub
 		public static function createOrganizer( $data, $post_status = 'publish' ) {
 			if ( ( isset( $data['Organizer'] ) && $data['Organizer'] ) || self::someOrganizerDataSet( $data ) ) {
 
+				$organizer_label = tribe_get_organizer_label_singular();
+
 				$postdata = array(
-					'post_title'  => $data['Organizer'] ? $data['Organizer'] : "Unnamed Organizer",
+					'post_title'  => $data['Organizer'] ? $data['Organizer'] : sprintf( __( 'Unnamed %s', 'tribe-events-calendar' ), ucfirst( $organizer_label ) ),
```
Comment (zbtirrell) [inline src/Tribe/API.php]: we changed the text domain since you made this pull request.  it is now `the-events-calendar`

**Pull request merged**